java.util.concurrent.RejectedExecutionException: Task java.util.concurrent.FutureTask@4cfa1dbf[Not completed, task = java.util.concurrent.Executors$RunnableAdapter@661655c9[Wrapped task = Thread[Thread-1570266,5,main]]] rejected from java.util.concurrent.ThreadPoolExecutor@43d4930e[Shutting down, pool size = 2, active threads = 0, queued tasks = 0, completed tasks = 1570263]
	at java.base/java.util.concurrent.ThreadPoolExecutor$AbortPolicy.rejectedExecution(ThreadPoolExecutor.java:2070)
	at java.base/java.util.concurrent.ThreadPoolExecutor.reject(ThreadPoolExecutor.java:833)
	at java.base/java.util.concurrent.ThreadPoolExecutor.execute(ThreadPoolExecutor.java:1365)
	at java.base/java.util.concurrent.AbstractExecutorService.submit(AbstractExecutorService.java:123)
	at cs451.links.FairLossLinks.send(FairLossLinks.java:52)
	at cs451.links.StubbornLinks.lambda$sendMessagesToBeSent$2(StubbornLinks.java:106)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
	at cs451.links.StubbornLinks.sendMessagesToBeSent(StubbornLinks.java:98)
	at cs451.links.StubbornLinks.lambda$new$1(StubbornLinks.java:78)
	at java.base/java.lang.Thread.run(Thread.java:833)
